src/App.tsx

```
@@ -57,7 +57,9 @@ const OrganizationFunds = lazy(
 );
 const OrganizationTransactions = lazy(
   () =>
-    import('screens/AdminPortal/OrganizationTransactions/OrganizationTransactions'),
+    import(
+      'screens/AdminPortal/OrganizationTransactions/OrganizationTransactions'
+    ),
```

ESLint

[error] Replace `⏎······'screens/AdminPortal/OrganizationTransactions/OrganizationTransactions'⏎····` with `'screens/AdminPortal/OrganizationTransactions/OrganizationTransactions'`

(prettier/prettier)